io.github.FreeDM.json

```diff
@@ -0,0 +1,106 @@
+{
+	"app-id": "io.github.FreeDM",
+	"runtime": "org.freedesktop.Platform",
+	"sdk": "org.freedesktop.Sdk",
+	"runtime-version": "1.6",
+	"command": "gzdoom.sh",
+	"rename-desktop-file": "freedm.desktop",
+	"rename-appdata-file": "freedm.appdata.xml",
+	"rename-icon": "freedm",
+	"finish-args": [
+		"--device=dri",
+		"--socket=wayland",
+		"--socket=x11",
+		"--share=ipc",
+		"--share=network",
+		"--socket=pulseaudio",
+		"--env=DOOMWADDIR=/app/share/games/doom"
+	],
+	"modules": [
+		{
+			"name": "fluidsynth",
+			"buildsystem": "cmake",
+			"config-opts": [
+				"-DLIB_SUFFIX="
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/FluidSynth/fluidsynth/archive/v1.1.9.tar.gz",
+					"sha256": "dd6321e13a7c875ef3032644bd3197e84b3d24928e2379bc8066b7cace7bd410"
+				}
+			]
+		},
+		{
+			"name": "gzdoom",
+			"buildsystem": "cmake",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/coelckers/gzdoom/archive/g3.2.5.tar.gz",
+					"sha256": "e9cf0aa5b7ee0b165532dee55e4965f6aabf1acadb79f7372f8e362540206748"
+				},
+				{
+					"type": "file",
+					"url": "https://github.com/coelckers/gzdoom/raw/master/soundfont/gzdoom.sf2",
```

Comment:
This soundfont is currently not being installed when building GZdoom from source. Upstream feature request: https://forum.zdoom.org/viewtopic.php?f=15&t=59182&p=1036858#p1036858